kraken-build/src/kraken/std/python/buildsystem/poetry_test.py

```diff
@@ -1,4 +1,6 @@
 from kraken.common.toml import TomlFile
+from kraken.std.python.pyproject import PyprojectHandler
+
 from kraken.std.python.buildsystem.poetry import PoetryPyprojectHandler
 from kraken.std.python.pyproject import PackageIndex
```

Comment:
```suggestion

from kraken.std.python.buildsystem.poetry import PoetryPyprojectHandler
from kraken.std.python.pyproject import PackageIndex, PyprojectHandler
```
(we should use `ruff check --fix` to automate this: https://docs.astral.sh/ruff/formatter/#sorting-imports)

Reply:
That is a bit annoying ...

Reply:
I've opened https://github.com/kraken-build/kraken/pull/298/ to automate this.